docs/application/dotnet/guides/text-input/voice-control-manager.md

```diff
@@ -0,0 +1,1084 @@
+    }
+
+    void SetUnsetConversationRequestedCb()
+    {
+        try
+        {
+            /// Set the callback
+            VoiceControlManagerClient.ConversationRequested += conversationRequested;
+
+            /// Unset the callback
+            VoiceControlManagerClient.ConversationRequested -= conversationRequested;
+        }
+        catch (Exception e)
+        {
+            /// Error handling
+        }
+    }
+    ```
+
+-   Set the private data updated callback invoked when voice control engine set private data to voice control manager:
+    ```
+    /// callback
+    void privateDataUpdated(object sender, PrivateDataUpdatedEventArgs e)
+    {
+        /// Your code
+    }
+
+    void SetUnsetPrivateDataUpdatedCb()
+    {
+        try
+        {
+            /// Set the callback
+            VoiceControlManagerClient.PrivateDataUpdated += privateDataUpdated;
+
+            /// Unset the callback
+            VoiceControlManagerClient.PrivateDataUpdated -= privateDataUpdated;
+        }
+        catch (Exception e)
+        {
+            /// Error handling
+        }
+    }
+    ```
+
+
+-   Set the specific engine result callback invoked when voice control engine set specific engine's result to voice control manager:
+
+    ```
+    /// callback
+    void specificEngineResult(object sender, SpecificEngineResultEventArgs e)
+    {
+        /// Your code
+    }
+
+    void SetUnsetSpecificEngineResultCb()
+    {
+        try
+        {
+            /// Set the callback
+            VoiceControlManagerClient.SpecificEngineResult += specificEngineResult;
+
+            /// Unset the callback
+            VoiceControlManagerClient.SpecificEngineResult -= specificEngineResult;
+        }
+        catch (Exception e)
+        {
+            /// Error handling
+        }
+    }
+    ```
+
+
+-   Set the pre recognition result updated callback invoked when voice control engine set the pre recognition results(partial ASR) to voice control manager.
+
+    You can get the results of pre result event type and the pre recognition results, the partial ASR.
+
+    ```
+    /// callback
+    void preRecognitionResultUpdated(object sender, PreRecognitionResultUpdatedEventArgs e)
+    {
+        /// Your code
+    }
+
+    void SetUnsetPreRecognitionResultUpdatedCb()
+    {
+        try
+        {
+            /// Set the callback
+            VoiceControlManagerClient.PreRecognitionResultUpdated += preRecognitionResultUpdated;
+
+            /// Unset the callback
+            VoiceControlManagerClient.PreRecognitionResultUpdated -= preRecognitionResultUpdated;
+        }
+        catch (Exception e)
+        {
+            /// Error handling
+        }
+    }
+    ```
+
+
+-   Set the all recognition result received callback invoked when voice control engine set the all recognition result received to voice control manager.
+
+    You can get the recognized result, recognized text, and engine message results in this callback.
```

Comment:
Referencing https://github.com/Samsung/tizen-docs/pull/971#discussion_r392078513


=> To my understand, it sounds like

Set the all recognition result received callback invoked when voice control engine sends the all recognition result to voice control manager.
In the callback, the recognized result, recognized text, and engine message results are included and sent to all voice control clients.

@sungraejo and @rishu-samsung would you please check the sentence?

Reply:
@rishu-samsung 
please check it.

```
You can get the recognized result, recognized text, and engine message results in this callback.
```
to
```
Set the all recognition result received callback invoked when voice control engine sends the all recognition result to voice control manager.

In the callback, the recognized result, recognized text, and engine message results are included and recognized result can include more than two voice commands if two voice commands those have same command string are registered by each voice control clients.

If you want to select specific command in the recognized result, you can use SetRecognizedCommandsSelectionDelegate() method.
```